tests/fv3_conf/cpld_control_run.IN

```diff
@@ -44,7 +44,8 @@ cp    @[INPUTDATA_ROOT]/CICE_FIX/@[OCNRES]/mesh.mx@[OCNRES].nc .
 
 # WW3 fix/input
 if [[ $CPLWAV == .true. ]]; then
-  cp    @[INPUTDATA_ROOT_WW3]/mod_def.* .
+  cp    @[INPUTDATA_ROOT_WW3]/mod_def.points .
```

Comment:
If we are using ww3_shel, you do not need the mod_def.points

Reply:
I'm adding a test ``cpld_multigrid_p8`` which runs with the multigrid=true setting and compares to the ``cpld_control_p8`` baseline. I believe I still the mod_def.points if I'm running the multigrid version?

I can put in an additional if statement that says whether to copy the mod_def.points based on whether that option. I did change this section it from the current usage, which copies all mod_def files into the run directory, even though only one is actually needed.

Reply:
It'd be great to have a test with the multigrid option.  It'd actually make workflow changes easier if we used the ww3_multi.inp option in P8, but that's probably not a reason to choose multi.inp over shel.inp as it's not a ton of work to use shel.inp.   

Am I correct that the ATMW tests still use wmesmf? If so, we need to have say "points" and "pointswm", because depending on which set of switches you use we should have two different mod_def.points files for the two caps.   That being said, since we actually don't have multiple grids for the S2SW we could turn off the unified points or use the mx* mod_def and instead of writing "points" we put "ww3" for this line: https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/parm/ww3_multi.inp.IN#L9 and then we don't have to have multiple "points" mod_defs and we would still be testing the unified points part of the code (or as much as you can test that when you only have 1 grid).  

If that doesn't make sense maybe we just set up a quick meeting instead of trying to type everything?

Reply:
I'm not changing the atmw test at all. It uses wmesmf and the control_run.IN. I have a conditional in ``run_test.sh`` which makes ensures that the multi input is used when ``multigrid=false`` (the default). 

Today is a bit crazy trying to get the j=1 PR finished off. Let me get those started and I'll try to ping you.